src/components/Toggle/Toggle.js

```diff
@@ -69,18 +69,30 @@ const toggleWrapperNoMarginStyles = ({ noMargin }) =>
     margin-bottom: 0;
   `;
 
+const toggleWrapperRTLStyles = ({ theme, rtl }) =>
+  rtl &&
+  css`
+    flex-direction: row-reverse;
```

Comment:
I think you're missing something here to achieve the design you shared with me on Slack.
```suggestion
    flex-direction: row-reverse;
    justify-content: space-between;
```

Reply:
No need to spread the space as in the design the switch and the text are shown together.
The image I sent on Slack was just a use case how the Toggle looks like natively on iOS.

Reply:
> No need to spread the space as in the design the switch and the text are shown together.

Looking at the design file you linked, the toggle and label are shown on the same line, but with space in between.

_Target design_

![2step verification](https://user-images.githubusercontent.com/11017722/79835284-ba37f900-83ae-11ea-845c-b450c7728dba.png)

_Current implementation_

![2step verification squished](https://user-images.githubusercontent.com/11017722/79835292-be641680-83ae-11ea-88f3-360aa5c47466.png)

Reply:
Correct, you are absolutely right! Will push it now.